nucypher/blockchain/eth/registry.py

```diff
@@ -233,11 +237,13 @@ def read(self) -> Union[list, dict]:
         raise NotImplementedError
 
     @classmethod
-    def from_latest_publication(cls,
-                                *args,
-                                source_manager=None,
-                                network: str = NetworksInventory.DEFAULT,
-                                **kwargs) -> 'BaseContractRegistry':
+    def from_latest_publication(
+        cls,
+        *args,
+        source_manager=None,
+        network: str = NetworksInventory.DEFAULT_DOMAIN_NAME,
```

Comment:
I suggest removing the default here and always explicitly passing the correct value.  Is the default even used anywhere?

Reply:
👍 . This will get removed as part of the rebase over your changes.